rotkehlchen/tests/fixtures/greenlets.py

```diff
@@ -3,6 +3,6 @@
 from rotkehlchen.greenlets.manager import GreenletManager
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture
```

Comment:
Put this out of the PR. And probably what would make sense here is as you also hinted to in discord:

to just yield and then call greenlet_manager.clear(). As this is what clears the tasks.